opm/autodiff/FlowMainEbos.hpp

```diff
@@ -707,8 +700,17 @@ namespace Opm
         Grid& grid()
         { return ebosSimulator_->gridManager().grid(); }
 
+        Problem& ebosProblem()
```

Comment:
If it's not needed now it should be removed. It is better to have the compiler complaining "now you are changing the Problem" at some point in the future than to let it happen without notice.